lib/bridge.js

```diff
@@ -964,29 +969,33 @@ function queueAlgorithm(event) {
 }
 
 class EventQueue {
-    constructor(opt, consumeFn) {
-        this.type = opt.type;
+    constructor(type, consumeFn) {
+        this.type = type;
         this._queues = {
             // $identifier: {
-            //  events: [ {data: promise: } ],
+            //  events: [ {dataReady: } ],
             //  consuming: true|false
             // }
         };
         this.consumeFn = consumeFn;
     }
 
-    push(event, data, promise) {
-        var identifier = this.type === "per_room" ? event.room_id : "none";
+    push(event, dataReady) {
```

Comment:
Same for each of these functions, a [JSDoc](https://en.wikipedia.org/wiki/JSDoc) line would go a long way :)